_FIPS201/system.md

```diff
@@ -125,8 +125,8 @@ program) to which the cardholder wants to gain access.
 [^DPC]: The cardholder may authenticate with the PIV Card or a derived PIV credential.
 
 The relying subsystem depends on authorization mechanisms that define the privileges (authorizations)
-possessed by entities requesting to access a particular logical or physical resource. An example of this is
-an Access Control List (ACL) associated with a file on a computer system.
+possessed by entities requesting to access a particular logical or physical resource. Examples of this include
```

Comment:
SP 800-116 uses term "control panel" maybe stick with that term instead of control portal?